src/CMakeLists.txt

```diff
@@ -38,7 +35,10 @@ target_compile_options(${PROJECT_NAME} PRIVATE
   -Wall -Wextra -Wpedantic>
```

Comment:
These should probably be removed in favor of the other settings in compiler-flags.cmake.